cmd/tetragon/flags.go

```diff
@@ -77,6 +77,8 @@ const (
 	keyEnablePidSetFilter = "enable-pid-set-filter"
 
 	keyEnableMsgHandlingLatency = "enable-msg-handling-latency"
+
+	keyEnableDataEventsSizeMetric = "enable-data-events-size-metric"
```

Comment:
Is this flag needed?

From what I see, currently from all metrics only `LatencyStats` has to be explicitly enabled with a flag. Having a way to enable/disable metrics is useful, but having a separate flag for each of them doesn't seem scalable. If it's not necessary right now, then I would rather develop a more generic way to enable/disable metrics in the near future.

Reply:
any idea how bad/slow is the histogram observe call? I got the impression it's better if it's disabled by default

Reply:
I'm ok with this the main problem with some of the metrics was cardinality of the labels. This doesn't look bad from that side.

Reply:
ok, I removed the option